database/index.js

```diff
@@ -6,6 +6,12 @@ const hstore = require('pg-hstore');
 const db = new Sequelize('neighborhood', 'root', 'hrnyc18',
   {
     dialect: 'postgres',
+    pool: {
+      max: 10,
+      min: 1,
+      idle: 10000
+    },
+    logging: false
```

Comment:
any thoughts on logging? as a team we should probably explore and discuss this

Reply:
this is to stop overly-comprehensive logging from postgres, but totally agree we should discuss logging as a team for proxies

Reply:
discussing logs, we all want to align what service we're using
maybe loggly, free 30 day trial
data dog, 14 days free, multiple servers

likely won't get to this until after the holidays

https://www.loggly.com/signup/

https://www.datadoghq.com/